fastlane/lib/fastlane/actions/increment_build_number.rb

```diff
@@ -47,7 +47,7 @@ def self.run(params)
         end
 
         # Store the new number in the shared hash
-        build_number = Actions.sh("#{command_prefix} agvtool what-version", log: false).split("\n").last.strip
+        build_number = params[:build_number] ? params[:build_number].to_s.strip : Actions.sh("#{command_prefix} agvtool what-version", log: false).split("\n").last.strip
```

Comment:
Can we find a way to clean this line up a bit? Its a little bit long 🙃 Maybe something like...

```suggestion
        build_number = params[:build_number].to_s.strip
        build_number = Actions.sh("#{command_prefix} agvtool what-version", log: false).split("\n").last.strip if build_number.to_s == ""
```
⚠️ NOTE: I did not test this ^^^

Reply:
Works as expected, thanks! 💪